grass7/imagery/i.gabor/i.gabor.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+
+############################################################################
+#
+# MODULE:       i.gabor
+#
+# AUTHOR(S):    Owen Smith <ocsmit@protonmail.com>
+#
+# PURPOSE:      Compute and convolve Gabor filter banks and for spatial
+#               imagery.
+#
+# COPYRIGHT:    (C) 2021 by Owen Smith and the GRASS Development Team
+#
+#               This program is free software under the GNU General
+#               Public License (>=v2). Read the file COPYING that
+#               comes with GRASS for details.
+#
+############################################################################
+
+# %module
+# % description: Creates Gabor filter bank for an n-dimensional image
```

Comment:
@ocsmit `n-dimensional` Maybe, it's a typo? Can it really handle an arbitrary number of dimensions? There is only `gabor2d()` in the code. Did you mean `2-dimensional` or can we use it for 3D voxel segmentation, for example? If not, I would change it to `2-dimensional`.

Reply:
Good catch,  meant `2-dimensional`.